### DISCUSSION TITLE
How do I add Geosphere Austria as a weather source?

### DISCUSSION THREAD

**Discussion by fabian-st** [breezy-weather/breezy-weather#1294] [Q&A] [answered]
I saw that there was some activity regarding the support for GeoSphere Austria but I couldn't find a definite guide on how to use it as a source.

**Comment by papjul** [comment, marked as answer]
1. Make sure you run the latest version of Breezy Weather
2. Make sure your location is within the areas covered by GeoSphere Austria (Austria is fully covered but only small parts of nearby countries are covered)
3. Tap the pencil icon on top right of that location
4. Tap Weather sources
5. Choose GeoSphere Austria wherever you want to use it as a source and wherever it is supported
